src/App.js

```diff
@@ -7,6 +7,7 @@ import {
   PrivateRoute,
   Profile,
   Cart,
+  Wishlist,
```

Comment:
Must Change: There shouldn't be more than one same product in the wishlist page.

![image](https://user-images.githubusercontent.com/91944618/160813906-48861f6d-3a4a-435b-aa6e-eb3a1752f7f7.png)